src/site/stages/build/process-cms-exports/schemas/transformed/paragraph-situation_update.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+	type: 'object',
+	properties: {
+		contentModelType: { type: 'string', enum: ['paragraph-situation_update'] },
+		entity: {
+			type: 'object',
+			properties: {
+				entityType: { type: 'string' },
+				entityBundle: { type: 'string' },
+				//fieldDateAndTime: { type: 'string' },
+				fieldDateAndTime: {
+					type: 'object',
+					// These properties are strings resembling dates
+					properties: {
+						date: { type: 'string' },
+						value: { type: 'string' },
+					},
+				},
+				fieldSendEmailToSubscribers: { type: ['boolean'] },
+				fieldWysiwyg: { type: ['string'] },
```

Comment:
Usually these are like `{ processed: 'wysiwyg content here' }`. Is that not the case for this one?